lib/environment_manager.rb

```diff
@@ -37,7 +37,7 @@ def conf
       raise "no '#{key}' environment configuration found" if ! env_class || env_class.empty?
 
       env_opts[:key] = key # lets have it for a reference
-      return BushSlicer.const_get(env_class).new(env_opts)
+      return BushSlicer.const_get(env_class).new(**env_opts)
```

Comment:
why do we need this change?

Reply:
@akostadinov because w/o the change we will see this warning  `warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call`